element_electrode_localization/coordinate_framework.py

```diff
@@ -12,16 +12,27 @@
 schema = dj.schema()
 
 
-def activate(schema_name, *, create_schema=True, create_tables=True):
+def activate(
+    schema_name: str,
+    *,
+    create_schema: bool = True,
+    create_tables: bool = True,
+):
     """Activates the schema.
 
     Args:
-        schema_name (str): A string containing the name of the probe schema.
-        create_schema (bool): If True, schema will be created in the database.
-        create_tables (bool): If True, tables related to the schema will be created in the database.
+        schema_name (str): schema name on the database server
+        create_schema (bool): when True (default), create schema in the database if it
+                            does not yet exist.
+        create_tables (bool): when True (default), create schema tables in the database
+                             if they do not yet exist.
```

Comment:
```suggestion
        schema_name (str): A string containing the name of the probe schema.
        create_schema (bool): If True, schema will be created in the database.
        create_tables (bool): If True, tables related to the schema will be created in the database.
```
The old versions are fine here because they're a little more specific for `schema_name` and succinct for the rest.